runtime/simx/vortex.cpp

```diff
@@ -415,6 +437,21 @@ class vx_device {
       return 1;
     }
 
+    // HW: virtual mem allocator has the same address range as global_mem. next step is to adjust it
+    virtual_mem_ = new MemoryAllocator(ALLOC_BASE_ADDR, (GLOBAL_MEM_SIZE - ALLOC_BASE_ADDR), MEM_PAGE_SIZE, CACHE_BLOCK_SIZE);
+    CHECK_ERR(virtual_mem_reserve(PAGE_TABLE_BASE_ADDR, (GLOBAL_MEM_SIZE - PAGE_TABLE_BASE_ADDR), VX_MEM_READ_WRITE), {
+      return err;
+    });
```

Comment:
I'm assuming when you adjust the memory range you will remove this from the range?
Or are you planning to remove the `page_table_mem` allocator?

Reply:
Yup, It returns the `page_table_mem` memory range to the system, and `page_table_mem` is automatically deleted when vx_device object is destructed.